internal/payloads/pubkey_payload.go

```diff
@@ -8,32 +8,54 @@ import (
 	"github.com/go-chi/render"
 )
 
-type PubkeyPayload struct {
-	*models.Pubkey
+// See models.Pubkey
+type PubkeyRequest struct {
+	Name string `json:"name" yaml:"name"`
+	Body string `json:"body" yaml:"body"`
+}
+
+// See models.Pubkey
+type PubkeyResponse struct {
+	ID                int64  `json:"id,omitempty" yaml:"id,omitempty"`
+	AccountID         int64  `json:"-" yaml:"-"`
+	Name              string `json:"name" yaml:"name"`
+	Body              string `json:"body" yaml:"body"`
+	Type              string `json:"type,omitempty" yaml:"type,omitempty"`
+	Fingerprint       string `json:"fingerprint,omitempty" yaml:"fingerprint,omitempty"`
+	FingerprintLegacy string `json:"fingerprint_legacy,omitempty" yaml:"fingerprint_legacy,omitempty"`
 }
-type (
-	PubkeyRequest  PubkeyPayload
-	PubkeyResponse PubkeyPayload
-)
 
 func (p *PubkeyRequest) Bind(_ *http.Request) error {
-	// Fingerprint is read-only field
-	p.Fingerprint = ""
 	return nil
 }
 
 func (p *PubkeyResponse) Render(_ http.ResponseWriter, _ *http.Request) error {
 	return nil
 }
 
+func (p *PubkeyRequest) Model() *models.Pubkey {
```

Comment:
I am not sure if Model is self-explanatory enough. The package is payloads, so we'd call this payloads.Model from outside, right? As this is exported, someone might call this from other package.

Reply:
This is a function with receiver so the use is:

```go
pr := PubkeyRequest{}
// ...
pubkeyModel := pr.Model()
```

It converts payload to model. Recommend me a better name, I did not like it either :-)

Reply:
As I think about it, this might be the best way. I just found it confusing, that the word 'model' can be a noun, verb, or adjective. How about something like "NewModel"?

Reply:
Maybe `ToModel` or Ruby on Rails-like `AsModel`?

Reply:
Going for the latter.